### PR TITLE
[QPE-583] avoid negatives

### PR DESCRIPTION
**Pull request by ahmed-bazzara**
Jira: https://tretton37.atlassian.net/browse/QPE-583

### CODE REVIEW

**Comment by flennmark** [comment]
Looks ok to me. Please merge after code review

**Comment by johnlunde** [comment]
@ahmed-Bazzara : Please solve the conflicts :)

**Comment by muenva** [comment]
Not sure how to verify this as I have no context what was the issue and how to replicate it @Caele . So I've made 3 scenarios with setting the measures manually. Was it one of them or something else?

1) Measures: 10, -40
---
measures-10-negative-40


2) Measures: -40, 10
---
measures-negative-40-10


3) Measures: -40, -10
---
measures-negative-40-neagtive-10

File: src/ldw-barsPlus.js [hunk 1]
```diff
@@ -1248,7 +1263,25 @@ export default {
     var dim1 = g.data.map(function (d) { return d.dim1; });
     if (g.orientation == "H") dim1.reverse();
     g.dScale.domain(dim1);
-    g.mScale.domain([0, d3.max(g.data, function (d) { return (g.normalized ? 1 : d.offset) * g.gridHeight; })]);
+    var max = d3.max(g.data, function (d) {
```
Comment (kristofferlind) [inline src/ldw-barsPlus.js]: isn't this block of code identical to the one starting at 447? maybe you could define it once and just pass it in  in both those cases?

**Review by kristofferlind — CHANGES_REQUESTED** [review]

**Review by kristofferlind — APPROVED** [review]

**Pull request merged**